.ats/main.yaml

```diff
@@ -0,0 +1,38 @@
+app-tests-app-config-file: tests/test-values.yaml
+app-tests-deploy-namespace: loki
+
+smoke-tests-cluster-type: kind
+smoke-tests-cluster-config-file: tests/kind-config.yaml
+
+upgrade-tests-cluster-type: kind
+upgrade-tests-cluster-config-file: tests/kind-config.yaml
+upgrade-tests-app-catalog-url: https://giantswarm.github.io/giantswarm-catalog
+upgrade-tests-app-config-file: tests/test-values.yaml
+
+skip-steps: [functional]
+
+# --------
+# Uncomment values below this line to be able to execute app-test-suite against an externally created
```

Comment:
What are those comments?

Reply:
This is an additional config for ats to enable to run against externally created kind clusters (such as a local one)

Reply:
okay :)